rules/mapping.smk

```diff
@@ -54,15 +54,17 @@ rule remove_decoy:
         decoy = config["params"]["remove_decoy"]["bed"] 
     output: 
         out_rm = "decoy_rm/{sample}-{unit}.removed_reads.bam",
```

Comment:
Can you remove out_rm from this rule and the commnd since this bam isn't being used?